quickstart_template/301-jenkins-aptly-spinnaker-vmss.sh

```diff
@@ -163,11 +174,22 @@ echo "jenkins.model.Jenkins.instance.securityRealm.createAccount(\"$jenkins_user
 run_util_script "jenkins/run-cli-command.sh" -cif "addUser.groovy" -c "groovy ="
 rm "addUser.groovy"
 
+#Change the Jenkins port so it doesn’t conflict with the Spinnaker front50 port
+port=8082
+sed -i -e "s/\(HTTP_PORT=\).*/\1$port/"  /etc/default/jenkins
+service jenkins restart
+
+#service may failed to start for redis issue 
+sudo redis-server /etc/redis/redis.conf
```

Comment:
Is this a start or restart operation for Redis? What is redis issue here

Reply:
1. @michaeljqzq , this is to start redis. Currently using the Local Debian installation mode cannot success due to redis not being started. I'm not sure yet whether this is Azure specific issue but instead of digging into the root cause, manually starting the redis service could be a cheap way.

2. Please fix typo: "#service may fail(no 'ed') to start for the redis issue"

3. Please add a check logic to ensure whether the redis is started. (Its port is 6379), If not then start it, rather than directly starting it.